dash-pipeline/bmv2/stages/trusted_vni.p4

```diff
@@ -0,0 +1,46 @@
+#ifndef _DASH_STAGE_TRUSTED_VNI_P4_
+#define _DASH_STAGE_TRUSTED_VNI_P4_
+
+control trusted_vni_stage(
+    inout headers_t hdr,
+    inout metadata_t meta)
+{
+    action permit() {}
+
+    @SaiTable[single_match_priority = "true", api = "dash_trusted_vni", order=0, isobject="false"]
+    table global_trusted_vni {
+        key = {
+            meta.rx_encap.vni: range @SaiVal[name = "vni_range"];
+        }
+
+        actions = {
+            permit;
+        }
+    }
+
+    @SaiTable[single_match_priority = "true", api = "dash_trusted_vni", order=1]
+    table eni_trusted_vni {
+        key = {
+            meta.eni_id : exact @SaiVal[type="sai_object_id_t"];
+            meta.rx_encap.vni: range @SaiVal[name = "vni_range"];
+        }
+
+        actions = {
+            permit;
+            @defaultonly deny(meta);
+        }
+        const default_action = deny(meta);
+    }
+
+    apply {
+        if (global_trusted_vni.apply().hit) {
+            return;
+        }
+
+        if (!eni_trusted_vni.apply().hit) {
```

Comment:
better add an comment to make sure the pipeline can capture the behavior right.

Reply:
Have added TODO comment before `trusted_vni_stage.apply(hdr, meta)`.